### PR TITLE
Fixing inventory changes

### PR DESCRIPTION
**Pull request by knoxiiw**
Contact me anytime if you have questions etc.

### CODE REVIEW

File: rss-client/src/app/interfaces/user.ts [hunk 1]
```diff
@@ -2,9 +2,8 @@ export interface User {
   userId: number,
   email: string,
   password: string,
-  profilePic: string,
+  profilePic: any,
```
Comment (Bnbingham) [inline rss-client/src/app/interfaces/user.ts:5]: Shouldn't be a problem, but we'd like to know why this change is necessary.

File: rss-client/src/app/services/account.service.ts [hunk 2]
```diff
@@ -11,26 +11,9 @@ export class AccountService {
 
   mockAccounts: Account[];
```
Comment (Bnbingham) [inline rss-client/src/app/services/account.service.ts:13]: Why is anything in this test being deleted?

File: rss-client/src/app/services/user.service.ts [hunk 3]
```diff
@@ -33,11 +33,6 @@ export class UserService {
     headers: new HttpHeaders({ 'Content-Type': 'application/json' }),
   };
```
Comment (Bnbingham) [inline rss-client/src/app/services/user.service.ts:35]: Why is anything in here being changed?

File: rss-client/src/app/services/account.service.ts (hunk 2 above)
Comment (knoxiiw) [reply rss-client/src/app/services/account.service.ts:13]: Sorry, I was under the impression that this was an anomaly from my team. If it's actually something that you guys put in, then definitely keep it.

File: rss-client/src/app/services/user.service.ts (hunk 3 above)
Comment (knoxiiw) [reply rss-client/src/app/services/user.service.ts:35]: Again, I was under the impression that this was mock data that my team added and not yours. If these changes are unnecessary then 100% discard them

File: rss-client/src/app/interfaces/user.ts (hunk 1 above)
Comment (knoxiiw) [reply rss-client/src/app/interfaces/user.ts:5]: It's how it looked before my giant pull request. Just wanted to change it back.
